_posts/2015-09-24-agile-fluency.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "ePages turns agile"
+date: "2015-09-24 10:10:00"
+icon: refresh
+categories: agile
+authors: ["Anja B."]
+---
+
+Did you ever wonder how companies that are going the agile way are developing further over time?
+And how long it actually takes to get from level to level? Or how many companies achieve certain levels of agility?
+
+Here are all the answers:
+
+{% image blog-agile-fluency.png %}{% endimage %}
+
+The picture shows the so called Agile Fluency Model (originally released by Diana Larsen and James Shore, read [here](http://martinfowler.com/articles/agileFluency.html) for further information).
+The numbers are based on questions that had been asked to companies who changed to an agile methodology some years ago.
+The most impressive numbers are probably, that it takes really long to achieve two or three stars.
```

Comment:
The most impressive numbers are probably, that it takes a really long time to achieve two or three stars.